lib/global_id/locator.rb

```diff
@@ -37,13 +37,8 @@ def locate(gid, options = {})
       #   we will use #where(id: ids) instead, which does not raise on missing records.
       def locate_many(gids, options = {})
         if (allowed_gids = parse_allowed(gids, options[:only])).any?
-          models_and_ids  = allowed_gids.collect { |gid| [ gid.model_name.constantize, gid.model_id ] }
-          ids_by_model    = models_and_ids.group_by(&:first)
-          loaded_by_model = Hash[ids_by_model.map { |model, ids|
-            [ model, find_records(model, ids.map(&:last), ignore_missing: options[:ignore_missing]).index_by { |record| record.id.to_s } ]
-          }]
-
-          models_and_ids.collect { |(model, id)| loaded_by_model[model][id] }.compact
+          locator = locator_for(allowed_gids.first)
```

Comment:
Should we document `locate_many` is per app? It's implicit in `locate` (because you can't do it), but it might be a good idea to note it.